### ISSUE TITLE
OpenGL.Net.CoreProfile

### ISSUE THREAD

**Issue by pragmascript** [luca-piccioni/OpenGL.Net#114]
Hello,

I was trying getting a sample running with `OpenGL.Net.CoreProfile`. Unfortunately getting CoreProfile from NuGet doesn't seem to work. So I tried building it manually from Source. 
`OpenGL.Net.dll` builds fine also `OpenGL.Net.Math.dll` however I cannot seem to find a dll for OpenGL Core Profile. Is this dll deprecated?

**Comment by luca-piccioni** [comment]
No, the DLL is not deprecated. It is generated by the post-build script.

Can you be more specific about about "doesn't seem to work"?

**Comment by ForeverZer0** [comment]
Would be nice to have a .NET Core NuGet package. Currently the OpenGL.CoreProfile package only seems to support .NET Framework and Mono. As .NET Core has matured now, it is usually favored over those two frameworks. Being the compatibility profile can be built and has a package, I assume this is just a matter of configuring a project file and adding the package?

**Comment by LittleRainGames** [comment]
> Can you be more specific about about "doesn't seem to work"?

In the project references, there is a warning symbol on the CoreProfile.
When double clicking, it will not show you the contents, instead it gives the error message in quotes.
"This project cannot be viewed in the object browser because it is unavailable or not yet built. Please ensure that the project is available and built."
![coreProfileError](https://user-images.githubusercontent.com/27095679/61197420-b477ef00-a689-11e9-94cb-d757e0f0c3cf.png)

**Comment by ForeverZer0** [comment]
@LittleRainGames You appears you have both `OpenGL.NET` and `OpenGL.NET.CoreProfile.
